src/pages/Admin/Room/Edit/RoomEdit.tsx

```diff
@@ -134,24 +54,96 @@ const EditRoom: React.FC<EditRoomProps> = ({
 
       <Modal.Body>
         <Form onSubmit={handleSubmit(onSubmit)}>
-          {renderNameInput()}
-          {renderUrlInput()}
-          {renderDescriptionInput()}
-          {renderLogoInput()}
-
-          {roomTemplate?.customInputs?.map((input) => renderCustomInput(input))}
-
-          {renderTemplateSelect()}
```

Comment:
Technically all of this (+related) refactoring seems OOS for this PR as it's title/description/linked issue are concerned. What was the reasoning/thought process behind this? Does it relate to a separate issue? It's generally best to keep PRs focussed on one thing, as it make it easier to review + easier to see where the changes happened/why in future.

Reply:
I just thought that it's easier to read/write scss if it doesn't use the "renderSomeElement()" functions. For now, should I leave it or put everything back to how it was?

Reply:
Happy for you to leave it for now. We want to get rid of the 'render function' pattern in general anyway.

My comments were more related to understanding the 'why' of it, and to teach about keeping our PR's focussed. It turned what would have been a fairly quick tick/flick code review into something where I had to 'unpick' the changes, reverse engineer the intent, etc; to then be able to review it and see/make comments. Takes a lot longer/more effort/etc that way.